index.html

```diff
@@ -23,17 +23,26 @@
           <a href="#">Contact</a>
         </div>
     </nav>
-
+<!---------------Desktop header version--------------------->
+<nav class="nav-bar">
+  <ul>
+    <li><a href="#">Portfolio</a></li>
+    <li id="about"><a href="#">About</a></li>
+    <li><a href="#">Contact</a></li>
+  </ul>
+</nav>  
+<!------------- End of desktop header version---------------->
   </header>
-   
+  
     <section class="headline">
-      
-        <h2>I’m Ahmed Glad to see you!</h2>
+        
+        <h2>I’m Ahmed<br> Glad to see you!</h2>
 
         <p>I’m a software developer! I can help you build a product,
-             feature or website Look through some of my work and experience! 
-             If you like what you see and have a project you need coded,
-             don’t hestiate to contact me.</p>  
+           feature or website Look through some of my work and experience! 
+           If you like what you see and have a project you need coded,
+           don’t hestiate to contact me.
+        </p>  
 
             <div class="Social"> 
```

Comment:
Please make sure you did not introduce new bugs on your code while you are adding desktop view. Here, social links needs to be under the text like on the Figma template, not inside them. Also please add Medium and Angel.co links and icons for here and "About Myself" section.
<img width="473" alt="Screenshot 2023-01-26 at 1 51 05 AM" src="https://user-images.githubusercontent.com/25887253/214715518-2d0ee93b-a0e5-4c87-a2b5-5ac7ccef3419.png">